frontend_app_file/src/container/File.jsx

```diff
@@ -745,7 +747,7 @@ class File extends React.Component {
           customClass={`${state.config.slug}`}
           i18n={i18n}
         >
-          <div /* this div in display flex, justify-content space-between */>
+          <div> {/* this div in display flex, justify-content space-between */}
```

Comment:
was this firing a linting error ?

Reply:
Yes: react/jsx-no-comment-textnodes

https://github.com/standard/eslint-config-standard-jsx/blob/0fc0f43a8feaec180ecd0f258b03c081df9b47ef/eslintrc.json#L33

Reply:
Yes: react/jsx-tag-spacing

https://github.com/standard/eslint-config-standard-jsx/blob/0fc0f43a8feaec180ecd0f258b03c081df9b47ef/eslintrc.json#L39

Reply:
ok you can resolve